nncf/experimental/torch/search_building_blocks/search_graph.py

```diff
@@ -261,7 +263,8 @@ def get_merged_original_graph_with_pattern(orig_graph: nx.DiGraph, hw_fused_ops:
     if not hw_fused_ops:
         return merged_graph
     # pylint: disable=protected-access
-    pattern_fusing_graph = PT_HW_FUSED_PATTERNS.get_full_pattern_graph()
+    pattern_fusing_graph =\
+        PatternsManager().get_full_pattern_graph(BackendType.TORCH, TargetDevice.ANY)
```

Comment:
Can these patterns be applied only for ANY (CPU) device?

Reply:
Torch has no device specific logic in patterns

Reply:
> Torch has no device specific logic in patterns

Do you mean quantization in general or some specific algorithms that use this method?

Reply:
There are no algorithms in NNCF Torch that's using information about device when collecting fussing pattern. Fussing patterns are always the same

Reply:
@alexsu52, @AlexKoff88, is the statement above correct? Should PyTorch PTQ quantization use types of devices that differ from ANY?